runatlantis.io/docs/server-configuration.md

```diff
@@ -989,6 +989,21 @@ Setting this to `false` can be useful in an air-gapped environment where a downl
   ```
   A token for Terraform Cloud/Terraform Enterprise integration. See [Terraform Cloud](terraform-cloud.html) for more details.
 
+### `--use-tf-plugin-cache`
+```bash
+atlantis server --use-tf-plugin-cache=false
+# or
+ATLANTIS_USE_TF_PLUGIN_CACHE=false
+```
+Set to false if you want to disable terraform plugin cache.
+
+This flag is useful when having multiple projects that need to run a plan and apply in the same PR to avoid the race condition of `plugin_cache_dir` concurrently, this is a terraform known issue, more info:
```

Comment:
Is there any way this sentence could be made more clear? 

Specifically, does disable or enabling of this flag prevent the race condition? 

E.g - one has Atlantis with lots of parallel plans and applies. This flag is "useful" but how? Should it be set to false? If so, the wording should dictate that

Reply:
I am trying to figure this out right now 🫠